mlflow/utils/validation.py

```diff
@@ -98,6 +100,15 @@ def _validate_tag(key, value):
     _validate_length_limit("Tag value", MAX_TAG_VAL_LENGTH, value)
 
 
+def _validate_experiment_tag(key, value):
+    """
+    Check that a tag with the specified key & value is valid and raise an exception if it isn't.
+    """
+    _validate_tag_name(key)
+    _validate_length_limit("Tag key", MAX_EXPERIMENT_TAG_KEY_LENGTH, key)
```

Comment:
While we're at it, can we update the error message for `_validate_length_limit` (https://github.com/mlflow/mlflow/blob/067eea7ebb3bf9e0b0c164958eb0b619e3bf5119/mlflow/utils/validation.py#L122) to truncate the value emitted in the error message? Attempting to set a tag with a value of length 50,000 prints 50,000 characters to the console.

Reply:
Displaying the first 250 characters is probably sufficient